README.md

```diff
@@ -4,10 +4,13 @@
 
 ### `solx` is an optimizing compiler for Solidity
 
-The project aims to bring state-of-the-art optimizing compilers to Ethereum by:
-- Reducing gas consumption of EVM code.
-- Eliminating the need for inline assembly.
-- Providing industry-grade tools like an interactive debugger.
+It passes [our test suite](https://github.com/matter-labs/era-compiler-tests), which includes tests from the `solc` project and real-world projects. All `solc` tests are updated with each new `solc` release. Arbitrary contracts are expected to compile correctly, except for stack-too-deep issues. Benchmarks indicate that `solx` generates larger code than `solc` but reduces gas consumption on average.

```

Comment:
It would be nice to get an overview of what tests we are running (e.g. solidity compiler tests, uniswap, etc).

Reply:
Do you want to repeat this list from the tests repo?
[UniswapV2](https://github.com/matter-labs/era-compiler-tests/blob/main/solidity/complex/defi/UniswapV2Router01)
[UniswapV3](https://github.com/matter-labs/era-compiler-tests/blob/main/solidity/complex/defi/UniswapV3)
[StarkEx Verifier](https://github.com/matter-labs/era-compiler-tests/blob/main/solidity/complex/defi/starkex-verifier)
[Curve](https://github.com/matter-labs/era-compiler-tests/blob/main/vyper/complex/defi/Curve)
[Mooniswap](https://github.com/matter-labs/era-compiler-tests/blob/main/solidity/complex/defi/Mooniswap)
[SHIT](https://github.com/matter-labs/era-compiler-tests/blob/main/solidity/complex/defi/shitdao)

Reply:
Do I get it right that UniswapV3 is tested using non-statically linked `solc`? If so, we should probably explain it (otherwise it might be confusing)

Reply:
Ok. Then it probably makes sense to add that we are to extend real life contracts coverage (the idea we discussed privately).

Reply:
I mentioned UniswapV2 and Mooniswap. It's too complicated to explain about zksolc.
We can explain modularity and how our testing actually works later.